documentation/schema.json

```diff
@@ -1063,6 +1063,15 @@
           ],
           "description": "Enables or disables the exceptions feature. By default, this feature is enabled if in-process features are enabled.",
           "default": true
+        },
+        "TopLevelLimit": {
+          "type": [
+            "integer",
+            "null"
+          ],
+          "description": "The number of top-level exceptions to keep in the cache before automatically removing older exceptions. A top-level is exception is one that is not an inner exception of another exception.",
+          "format": "int32",
+          "default": 20
```

Comment:
Do we want to have max/min on this?

Reply:
Just noticed we do at least check for > 0 in `ConfiguredExceptionsStore.cs`, which was my primary concern.